java/client/src/main/java/glide/api/commands/ServerManagementCommands.java

```diff
@@ -133,4 +133,17 @@ public interface ServerManagementCommands {
      * }</pre>
      */
     CompletableFuture<String[]> time();
+
+    /**
+     * Synchronously saves the DataBase.
```

Comment:
The review checklist mentions the following, is it applicable here or no?
`For standalone keyless commands: Document whether the command is distributed to all nodes or only to the primary (e.g., ACL commands).`

Reply:
Cluster client implements `ServerManagementClusterCommands` interface, but this one goes for standalone client - routing is not applicable to it.
Please, review docs for cluster client.